http-disk-server

```diff
@@ -367,6 +443,10 @@ def run_server(disk, ip, port):
                 eprint('Failed to close server: {}.'.format(e))
             close_device(disk_fd)
```

Comment:
If this fails (or the eprint at previous line), you won't reach `emit_server_ready()`. Could this be an issue?

Reply:
It can fail the XCP-ng HA activation and/or attach at host boot. So I will add a finally close. Just because I don't see a problem now doesn't mean there won't be one later.